plugins/modules/ec2_launch_template.py

```diff
@@ -671,7 +686,9 @@ def main():
             options=dict(
                 http_endpoint=dict(choices=['enabled', 'disabled'], default='enabled'),
                 http_put_response_hop_limit=dict(type='int', default=1),
-                http_tokens=dict(choices=['optional', 'required'], default='optional')
+                http_tokens=dict(choices=['optional', 'required'], default='optional'),
+                http_protocol_ipv6=dict(choices=['disabled', 'enabled'], default='disabled'),
+                instance_metadata_tags=dict(choices=['disabled', 'enabled'], default='disabled'),
```

Comment:
Hm maybe we must pop `metadata_options.instance_metadata_tags` if current used boto3 version does not support it.  
Otherwise all playbook will fail using 3.1.0 when boto3 is to old.

Are there any examples how we check the boto3 version? @tremble  @alinabuzachis

Reply:
Ok, done. But it's failing because of `http_protocol_ipv6`  
No idea when it was introduced